src/test/kotlin/apply/AdministratorFixtures.kt

```diff
@@ -22,8 +23,17 @@ fun createAdministratorData(
     username: String = ADMINISTRATOR_USERNAME,
     password: String = ADMINISTRATOR_PASSWORD,
     confirmPassword: String = ADMINISTRATOR_PASSWORD
-): AdministratorData {
-    return AdministratorData(name, username, password, confirmPassword)
+): CreateAdministratorFormData {
+    return CreateAdministratorFormData(name, username, password, confirmPassword)
+}
+
+fun createAdministratorUpdateFormData(
+    id: Long = 0L,
+    name: String = "Admin",
+    password: String = "abcd12345",
+    confirmPassword: String = "abcd12345"
```

Comment:
c: 기존 코드를 많이 보고 분석해 보시기 바랍니다.

```suggestion
    name: String = ADMINISTRATOR_NAME,
    password: String = ADMINISTRATOR_PASSWORD,
    confirmPassword: String = ADMINISTRATOR_PASSWORD
```

Reply:
update fixture 부분도 `ADMINISTRATOR_NAME`, `ADMINISTRATOR_PASSWORD`로 할 경우, 테스트 코드를 작성할 때 정말 변경이 제대로 된 것인지 확인하기 어렵다고 생각됩니다. (변경 전과 변경 후가 동일한 값으로 지정되어 항상 테스트가 통과할 것 같다는 생각입니다.) 이런 이유로 위와 같이 작성했는데 제이슨 생각은 어떤가요?

Reply:
다른 방법으로 수정해봤는데 확인 부탁드립니다!